md/release-notes.md

```diff
@@ -61,9 +64,10 @@ When cloning a project, gain time by selecting the branches you want to clone in
 Add documentation to your pages by providing information on a widget (e.g. how to use it), thanks to the new Description field. Use this information to [generate documentation](release-notes.md#project-documentation).
  
 <a id="connector-archetype"/>
+
```

Comment:
a title must always have a blank line before,otherwise it display as normal text